workers/unity/Packages/com.improbable.gdk.gameobjectcreation/GameObjectCreationSystemHelper.cs

```diff
@@ -1,20 +1,26 @@
 using Improbable.Gdk.Core;
 using Unity.Entities;
+using UnityEngine;
 
 namespace Improbable.Gdk.GameObjectCreation
 {
     public static class GameObjectCreationSystemHelper
     {
-        public static void EnableStandardGameObjectCreation(World world)
+        public static void EnableStandardGameObjectCreation(World world, IEntityGameObjectCreator creator = null,
+            GameObject workerGameObject = null)
```

Comment:
It looks like users need to right now determine themselves, when to create/clean up their worker entity gameobject. Would it make more sense to extend the `IEntityGameObjectCreator` with OnConnected/OnDisconnected callbacks to give users a more consistent way to set up/clean up their worker entity gameobject?

Reply:
I did think of that, but decided against it because 1) it feels weird to have callback-style methods that would be called just once over the lifetime of the object, 2) that would enforce that the users must override all methods even if they don't need, which is annoying boilerplate. Not 100% convinced, but this seemed like the better option.

Reply:
An example in the WorkerConnectors would certainly help a bit with that

Reply:
As long as everyone agrees that this is a stop gap, I'm fine with what this PR does for now. Longer term, I'd like to see the GameObjectCreationSystemHelper + IEntityGameObjectCreator replaced with something that looks more like a dispatcher.